config_examples/config_full.example.json

```diff
@@ -172,7 +172,25 @@
         "jwt_secret_key": "somethingrandom",
         "CORS_origins": [],
         "username": "freqtrader",
-        "password": "SuperSecurePassword"
+        "password": "SuperSecurePassword",
+        "ws_token": "secret_ws_t0ken."
+    },
+    // The ExternalMessageConsumer config should only be enabled on an instance
+    // that listens to outside data from another instance. This should not be enabled
+    // in your producer of data.
+    "external_message_consumer": {
+        "enabled": false,
+        "producers": [
+          {
+            "name": "default",
+            "url": "ws://localhost:8081/api/v1/message/ws",
```

Comment:
would it make sense to remove complexity for the user here - by adding `api/v1/message/ws` ourselfs (so the setting is actually `ws://localhost:8081`)?

the url cannot really change anyway - so all users MUST have the same value here - which increases complexity (and therefore chance for errors) without good reason.

Reply:
Agreed, we can just accept `host` and `port` parameters in the config.